app/templates/index.html

```diff
@@ -16,7 +17,18 @@
 		<span class="col-md-2">Creator</span>
 		<span class="col-md-2">Time Left</span>
 	</div>
+
+	<div>
+	{% with messages = get_flashed_messages() %}
+    {% if messages %}
+    	{% for message in messages %}
+		<p>{{ message }}</p>
+        {% endfor %}
+    {% endif %}
+    {% endwith %}
 	{% for challenge in challenges %}
+	</div>
```

Comment:
I'm not exactly sure how and if its an issue for just my browser - but I encounter this formatting issue only on this branch.
![image](https://github.com/RedBlueCarrots/WikiPath/assets/144818607/8ebfae10-f657-46ac-9428-8e8cc7d39178)

Reply:
it is likely to do with your placement of for challenge in challenge, which is adding a /div everytime

Reply:
Fixed bug, `</div>` now outside of for loop